hydra-tui/src/Hydra/TUI.hs

```diff
@@ -432,6 +433,8 @@ handleNewTxEvent Client{sendInput, sk} CardanoClient{networkId} s = case s ^? he
 --
 -- View
 --
+timeFormatted :: (FormatTime t) => t -> String
+timeFormatted r = formatTime defaultTimeLocale "%Dd %Hh %Mm %Ss" r
```

Comment:
The module is currently structured (where possible) top-down, so `draw` would be the more general function and `timeFormatted` more special. Also we have multiple `renderXXX` or `drawXXX` functions, which would be (IMO) a better name for this (not stating the result, but what the function does)

Could: move it further down, after `draw`.
Could: rename this function to `renderTime`